### ISSUE TITLE
server: /_status/logs/:node_id endpoint ignores `Accept` header

### ISSUE THREAD

**Issue by maxlang** [cockroachdb/cockroach#8569]
When the api endpoint is called with `Accept: application/x-protobuf` I expect to receive content encoded as a protobuf. Instead the API always returns JSON.

Compare

```sh
curl -H "Accept: application/x-protobuf" http://localhost:8080/_status/logs/1
```

with the working endpoint

```sh
curl -H "Accept: application/x-protobuf" http://localhost:8080/_admin/v1/cluster
```

**Comment by tamird** [comment]
Ah damnit. I know what this is about.

**Comment by tamird** [comment]
This was fixed in #8576.
